packages/contracts/src/2.0.0/protocol/AssetProxyOwner/AssetProxyOwner.sol

```diff
@@ -40,7 +38,7 @@ contract AssetProxyOwner is
     modifier validRemoveAuthorizedAddressAtIndexTx(uint256 transactionId) {
         Transaction storage tx = transactions[transactionId];
         require(isAssetProxyRegistered[tx.destination]);
-        require(tx.data.readBytes4(0) == REMOVE_AUTHORIZED_ADDRESS_AT_INDEX_SELECTOR);
+        require(readBytes4(tx.data, 0) == REMOVE_AUTHORIZED_ADDRESS_AT_INDEX_SELECTOR);
```

Comment:
The using syntax exists in 0.4.10 but it would require a library for just the `readBytes4` function. Doesn't seem worth it. (no action)